BUILD

```diff
@@ -0,0 +1,38 @@
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library",
+    "swift_binary"
+)
+
+swift_library(
+    name = "SourceKittenFramework",
+    module_name = "SourceKittenFramework",
+    srcs = ["//Source:SourceKittenFrameworkSources"],
+    deps = [
+        "@com_github_drmohundro_SWXMLHash//:SWXMLHash",
+        "@com_github_jpsim_yams//:Yams",
+        "//Source:SourceKit",
+        "//Source:Clang_C"
+    ],
+    defines = ["SWIFT_PACKAGE"],
+    visibility = ["//visibility:public"]
+)
+
+swift_library(
+    name = "SourceKittenLib",
```

Comment:
Is it necessary to have this intermediate library that's only used in the executable binary below? Why not fold those together? That's what we do on the SwiftPM side of things, there's just the SourceKittenFramework library, then the sourcekitten CLI executable.